runtime/common/BaseRestRemoteClient.h

```diff
@@ -320,20 +334,20 @@ class BaseRemoteRestRuntimeClient : public cudaq::RemoteRuntimeClient {
       stateIrPayload1.entryPoint = kernelName1;
       stateIrPayload1.ir =
           constructKernelPayload(mlirContext, kernelName1, nullptr, args1,
-                                 argsSize1, /*startingArgIdx=*/0);
+                                 argsSize1, /*startingArgIdx=*/0, nullptr);
```

Comment:
Looks like this will always cause old synthesis to be used - are we implementing this later?

Reply:
It doesn't change the current hookup, other than to add this extra unused pointer argument.

The new path enters via the `streamlinedLaunchKernel` entry point.